app/components/mandatarissen/update-state.js

```diff
@@ -245,6 +245,13 @@ export default class MandatarissenUpdateState extends Component {
     this.newStatus = status;
   }
 
+  @action
+  updateRangorde(event) {
+    if (event && typeof event.preventDefault === 'function')
```

Comment:
please always use {} in ifs

Reply:
oeps, seems to be in a few places, I'll fix this in a different PR